virtual-desktop-enhancer.ahk

```diff
@@ -354,6 +358,20 @@ OpenDesktopManager() {
     Send #{Tab}
 }
 
+; Let the user change desktop names with a prompt, without having to edit the 'settings.ini'
+; file and reload the program.
+; The changes are temprorary (names will be overwritten by the default values of
+; 'settings.ini' when the program will be restarted.
+ChangeDesktopName() {
+    currentDesktopNumber := _GetCurrentDesktopNumber()
+    currentDesktopName := _GetDesktopName(currentDesktopNumber)
+    InputBox, newDesktopName, Change desktop name, Change the desktop name of desktop #%currentDesktopNumber%, , , , , , , , %currentDesktopName%
```

Comment:
I'm looking in the documentation, but it looks like the titlebar icon is bound to the tray icon...